doc/source/schemas/alleleAnnotationmethods.rst

```diff
@@ -0,0 +1,700 @@
+
+  A CallSet is a collection of calls that were generated by the same analysis of the same sample.
+
+.. avro:record:: Call
+
+  :field callSetName:
+    The name of the call set this variant call belongs to.
+      If this field is not present, the ordering of the call sets from a
+      `SearchCallSetsRequest` over this `VariantSet` is guaranteed to match
+      the ordering of the calls on this `Variant`.
+      The number of results will also be the same.
+  :type callSetName: null|string
+  :field callSetId:
+    The ID of the call set this variant call belongs to.
+    
+      If this field is not present, the ordering of the call sets from a
+      `SearchCallSetsRequest` over this `VariantSet` is guaranteed to match
+      the ordering of the calls on this `Variant`.
+      The number of results will also be the same.
+  :type callSetId: null|string
+  :field genotype:
+    The genotype of this variant call.
+    
+      A 0 value represents the reference allele of the associated `Variant`. Any
+      other value is a 1-based index into the alternate alleles of the associated
+      `Variant`.
+    
+      If a variant had a referenceBases field of "T", an alternateBases
+      value of ["A", "C"], and the genotype was [2, 1], that would mean the call
+      represented the heterozygous value "CA" for this variant. If the genotype
+      was instead [0, 1] the represented value would be "TA". Ordering of the
+      genotype values is important if the phaseset field is present.
+  :type genotype: array<int>
+  :field phaseset:
+    If this field is not null, this variant call's genotype ordering implies
+      the phase of the bases and is consistent with any other variant calls on
+      the same contig which have the same phaseset string.
+  :type phaseset: null|string
+  :field genotypeLikelihood:
+    The genotype likelihoods for this variant call. Each array entry
+      represents how likely a specific genotype is for this call as
+      log10(P(data | genotype)), analogous to the GL tag in the VCF spec. The
+      value ordering is defined by the GL tag in the VCF spec.
+  :type genotypeLikelihood: array<double>
+  :field info:
+    A map of additional variant call information.
+  :type info: map<array<string>>
+
+  A `Call` represents the determination of genotype with respect to a
+  particular `Variant`.
+  
+  It may include associated information such as quality
+  and phasing. For example, a call might assign a probability of 0.32 to
+  the occurrence of a SNP named rs1234 in a call set with the name NA12345.
+
+.. avro:record:: Variant
+
+  :field id:
+    The variant ID.
+  :type id: string
+  :field variantSetId:
+    The ID of the `VariantSet` this variant belongs to. This transitively defines
+      the `ReferenceSet` against which the `Variant` is to be interpreted.
+  :type variantSetId: string
+  :field names:
+    Names for the variant, for example a RefSNP ID.
+  :type names: array<string>
+  :field created:
```

Comment:
Time formats should all be ISO8601. See https://github.com/ga4gh/schemas/blob/metadata/doc/source/metadata_intro.rst for reference/explanation.
Also, attributes should be ```recordCreateTime``` and ```recordUpdateTime```, and the type ```string```.